### ISSUE TITLE
(feat): loki output plugin

### ISSUE THREAD

**Issue by lukasmrtvy** [bitnami/bitnami-docker-fluentd#16]
Would be nice to include also output plugin for Loki ( fluent-plugin-grafana-loki )

https://grafana.com/docs/loki/latest/clients/fluentd/

Thanks

**Comment by juan131** [comment]
Hi @lukasmrtvy 

This is the list of the default plugins included in the image at this very moment:

- fluent-plugin-concat
- fluent-plugin-detect-exceptions
- fluent-plugin-elasticsearch
- fluent-plugin-kafka
- fluent-plugin-kubernetes_metadata_filter
- fluent-plugin-multi-format-parser
- fluent-plugin-prometheus
- fluent-plugin-s3
- fluent-plugin-systemd

I did a test including `fluent-plugin-grafana-loki` and it adds around **62KB**, which is barely nothing. I don't see any cons on adding this plugin to the list above. @javsalgar what do you think?

In the meantime, there is a section in the README.md that explains how to add extra plugins in the image by building your own image, see:

- https://github.com/bitnami/bitnami-docker-fluentd#extend-this-image

**Comment by lukasmrtvy** [comment]
What about entrypoint.sh that will install plugins from `PLUGINS` env variable ? It makes sense also for logstash image.

**Comment by juan131** [comment]
Thanks for your feedback @lukasmrtvy 

This image is non-root by default. Therefore, it's highly likely to face issue when installing gems during the **entrypoint.sh**. On the other hand, it's not a good practice to overload containers during the initialization with this kind of actions.

**Comment by juan131** [comment]
HI @lukasmrtvy 

We just released a new image version (**1.11.2-debian-10-r35**) that includes this plugin by default. Could you give it a try?

```console
$ docker run --rm -it bitnami/fluentd:1.11.2-debian-10-r35 -- ls -la /opt/bitnami/fluentd/gems | grep loki
drwxrwxr-x 1 root root 4096 Sep 23 09:27 fluent-plugin-grafana-loki-1.2.14
```
